stable/wordpress/templates/deployment.yaml

```diff
@@ -13,6 +13,7 @@ spec:
     metadata:
       labels:
         app: {{ template "fullname" . }}
+        {{ template "mariadb.fullname" . }}-client: "true"
```

Comment:
Having the name in the label name seems odd. Did we do this somewhere else before, i.e. is this current best-practive? Can you think of an alternative?